app/src/main/res/values/strings.xml

```diff
@@ -283,15 +283,15 @@
     <string name="store">Store</string>
     <string name="running_apps">Running Apps</string>
     <string name="kill_process">Kill</string>
-    <string name="disable_background_run">Disable Background Run</string>
-    <string name="pid_and_ppid">Process ID: %1$d, Parent Process ID: %2$d</string>
-    <string name="memory_virtual_memory">Memory: %1$s, Virtual Memory: %2$s</string>
+    <string name="disable_background_run">Don\'t run in the background</string>
+    <string name="pid_and_ppid">Process ID: %1$d. Parent Process ID: %2$d.</string>
+    <string name="memory_virtual_memory">Memory: %1$s. Virtual Memory: %2$s.</string>
     <string name="user_with_id">User: <xliff:g id="user_id" example="0">%1$d</xliff:g></string>
     <string name="user_and_uid">User: <xliff:g id="user_name" example="root">%1$s</xliff:g> (<xliff:g id="user_id" example="0">%2$d</xliff:g>)</string>
     <string name="clear_data">Clear Data</string>
     <string name="backup_apk">Backup APK</string>
     <string name="backup_restore">Backup/Restore</string>
-    <string name="disable_background">Disable Background</string>
+    <string name="disable_background">Don\'t run in the background</string>
```

Comment:
"Disable background" means something else though.
"Prevent running in the background"?

Reply:
> "Prevent running in the background"?

This can be used, but it's too broad for a title. _Prevent background operation_ or _Prevent background run_ is much concise.